src/style.css

```diff
@@ -1,3 +1,76 @@
+:root {
+  --link-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+  line-height: 1;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.logo img {
+  width: 100%;
+  height: auto;
+}
+
+a {
+  text-transform: uppercase;
+  color: #000;
```

Comment:
The default link color is set to black (`#000`). Ensure this provides enough contrast against the background for readability and accessibility. Consider using a color contrast checker to verify this.